chapters/sparse_resources.md

```diff
@@ -0,0 +1,299 @@
+# Sparse Resources
+
+Vulkan [sparse resources](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#sparsememory) are a way to create `VkBuffer` and `VkImage` which can be bound non-contiguously to one or more `VkDeviceMemory` allocations. There are many aspects and features of sparse resources which the [spec](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#sparsememory-sparseresourcefeatures) does a good job explaining. As the [implementation guidelines](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#_sparse_resource_implementation_guidelines) point out, most implementations use sparse resources to expose a linear virtual address range of memory to the application while mapping each sparse block to physical pages when bound.
```

Comment:
can we pluralize "`VkBuffer` and `VkImage`"? `VkBuffer`s looks dumb, so maybe "`VkBuffer` and `VkImage` objects"?